zilencer/views.py

```diff
@@ -106,3 +114,66 @@ def remote_server_notify_push(request,  # type: HttpRequest
                                      badge=1, zulip=apns_payload)
 
     return json_success()
+
+def register_remote_server(request):
+    # type: (HttpRequest) -> HttpResponse
+    if request.method == 'POST':
+        form = RemoteServerRegistrationForm(request.POST)
+        if form.is_valid():
+            email = form.cleaned_data['email']
+            api_key = form.cleaned_data['api_key']
+            uuid = form.cleaned_data['uuid']
+            hostname = form.cleaned_data['hostname']
+            with transaction.atomic():
```

Comment:
ok.
Also, similar comments to below on putting the arguments all on one line, to make `git grep` more useful. I know some codebases use the one argument per line style, but Zulip I think has settled on the denser lines, and I do find it more helpful for grepping and fitting more stuff on each page.